deepmd/dpmodel/atomic_model/base_atomic_model.py

```
@@ -191,13 +195,13 @@ def forward_common_atomic(
         ret_dict = self.apply_out_stat(ret_dict, atype)
 
         # nf x nloc
-        atom_mask = ext_atom_mask[:, :nloc].astype(np.int32)
+        atom_mask = ext_atom_mask[:, :nloc].astype(xp.int32)
         if self.atom_excl is not None:
             atom_mask *= self.atom_excl.build_type_exclude_mask(atype)
 
         for kk in ret_dict.keys():
```

Ruff

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)